### DISCUSSION TITLE
Way to modify the tagging rules

### DISCUSSION THREAD

**Discussion by agauttam** [utmstack/UTMStack#724] [Q&A] [answered]
Hi,

I want to modify and write the new tagging rules to avoid the false positives but I am not getting any option under threatmanagement >> tagging rules.

Regards,
Arun

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @agauttam try with this -> https://docs.utmstack.com/UTMStackComponents/Threat%20Management/FalsePositive.html

Best regards
